lib/response.js

```diff
@@ -22,7 +22,7 @@ module.exports = (function () {
             this.handler.response = buildSpeechletResponse({
                 sessionAttributes: this.attributes,
                 output: getSSMLResponse(speechOutput),
-                reprompt: getSSMLResponse(repromptSpeech),
+                reprompt: getSSMLResponse(repromptSpeech || speechOutput),
```

Comment:
Thanks @bclement-amazon. When I first raised this it seemed to be a lot of unnecessary code duplication everywhere in all the tutorials. I don't know if the landscaped has moved with real world use? It would still be optional with this change, it's just you get the reprompt out the box?

It feels like not wanting a reprompt would be the edge case, that the dev should opt out of if needed?